src/core/data.jl

```diff
@@ -34,7 +55,14 @@ function _update_data!(data::Dict{String,<:Any}, new_data::Dict{String,<:Any})
 end
 
 "checks if a given network data is a multinetwork"
-ismultinetwork(data::Dict{String,<:Any}) = (haskey(data, "multinetwork") && data["multinetwork"] == true)
+function ismultinetwork(data::Dict{String,<:Any})
```

Comment:
I am not 100% sure, but I think doing the inline variant may provide some additional help to the Julia compiler.  I think the inline variant results in a search and replace in the code rather than a function call.

For this function which is called infrequently, it won't make a big difference one way or the other, so if you prefer the function version, that's fine by me.

Reply:
Recently, I've landed upon the opinion that inline functions should only be used when there are substantial performance gains to be made. I think that, for the most part, the gains in debuggability and profilability usually outweigh the performance gains for inlining small functions like this. If we end up determining via profiling that these functions are indeed a bottleneck, I will agree to inline them 😄.